core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -657,5 +658,16 @@ open class TileInfo {
             improvement = improvementObject.name
     }
 
+    private fun convertHillToTerrainFeature(){
+        if (baseTerrain == Constants.hill){
```

Comment:
Check that hill is really a terrain feature in the ruleset - otherwise mods that haven't yet made the conversion could be affected, and I don't know what the results would be

Reply:
should I also revert the changes I made to isHill()?
```
old
fun isHill() = baseTerrain == Constants.hill || terrainFeatures.contains(Constants.hill)
new
fun isHill() = terrainFeatures.contains(Constants.hill)
```